src/package.json

```diff
@@ -0,0 +1 @@
+{ "type": "commonjs" }
```

Comment:
Note: This let's us write commonjs in `src/` which is what all our transformations and scripts are using, while exporting ES Modules.